src/components/alert/scripts/build-dev.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+if [ -z "$VARIANT" ]; then VARIANT=''; fi
+if [ -z "$TEMPLATE_LANG" ]; then TEMPLATE_LANG='en'; fi
+
+echo "Building package for demo. Variant: '$VARIANT', language: '$TEMPLATE_LANG'."
+
+browserify -t [ redirectify --dir '$VARIANT' ] -t reactify dev/example.jsx -o dev/app.js
```

Comment:
this looks like it should be double quotes in a shell script around a variable, but I haven't tried it yet